pkg/client/cli/license.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+type licenseInfo struct {
+	outputFile string
+}
+
+func LicenseCommand() *cobra.Command {
+	li := &licenseInfo{}
+	cmd := &cobra.Command{
+		Use:  "license",
+		Args: cobra.MinimumNArgs(1),
+
+		Short: "Get License from Ambassador Cloud",
+		Long:  "Get License from Ambasssador Cloud",
+		RunE:  li.getCloudLicense,
+	}
+	flags := cmd.Flags()
+
+	flags.StringVarP(&li.outputFile, "output-file", "f", "/tmp/license", "The file where you want the license secret to be output to")
+	return cmd
+}
+
+func (li *licenseInfo) getCloudLicense(cmd *cobra.Command, args []string) error {
+	outputFile := strings.TrimSpace(li.outputFile)
+	id := strings.TrimSpace(args[0])
```

Comment:
Why `TrimSpace`? I wouldn't expect there to be extra spaces here.

Reply:
Was just being overly cautious, but agreed it is unnecessary